src/providers/Web3Provider.tsx

```diff
@@ -96,10 +96,10 @@ export const Web3ContextProvider: FC<PropsWithChildren> = ({ children }) => {
   }
 
   const _removeEventListeners = (ethProvider = window.ethereum!) => {
-    ethProvider.off('accountsChanged', _accountsChanged)
-    ethProvider.off('chainChanged', _chainChanged)
-    ethProvider.off('connect', _connect)
-    ethProvider.off('disconnect', _disconnect)
+    ethProvider.off?.('accountsChanged', _accountsChanged)
```

Comment:
I don't think it's safe to simply *not* unsubscribe. I think safe options are:
- rewrite listeners so unsubscribing is never needed
- implement unsubscribing inside listeners
- reload page if can't unsubscribe

Reply:
Right, didn't want to spend much time on it. Since WalletConnect integration is not decided yet. We can just add Brave as a wallet in WalletConnect options.